scripts/install-for-ci.sh

```diff
@@ -1,7 +1,13 @@
 #!/bin/sh
-fetch
 set -e
 
+if [ "x$TRAVIS" = "xtrue" -a "x$ROSWELL_INSTALL_DIR" = "x" ]; then
+    ROSWELL_INSTALL_DIR=$HOME/usr
+    PATH=$HOME/usr/bin:$PATH
```

Comment:
You are right. It does not really make sense to do that, Travis does not have a proper user-writable directory in the PATH. I'll remove those lines from the PR.